dbms/src/Flash/Mpp/HashBaseWriterHelper.cpp

```diff
@@ -92,7 +94,6 @@ void computeHash(
     if unlikely (rows == 0)
         return;
 
-    hash.getData().resize(rows);
```

Comment:
why remove this

Reply:
Because `hash.getData().reset()` will call `PODArray::assign()`, which will `reset()`.